invenio_assets/webpack.py

```diff
@@ -17,13 +17,24 @@
 from markupsafe import Markup
 from pywebpack import ManifestEntry, UnsupportedExtensionError, bundles_from_entry_point
 
-project = WebpackBundleProject(
+webpack_project = WebpackBundleProject(
     __name__,
     project_folder="assets",
     config_path="build/config.json",
     bundles=bundles_from_entry_point("invenio_assets.webpack"),
 )
 
+rspack_project = WebpackBundleProject(
+    __name__,
+    project_folder="assets",
+    config_path="build/config.json",
+    bundles=bundles_from_entry_point("invenio_assets.webpack"),
+)
+rspack_project._package_json_source_path = "rspack-package.json"
```

Comment:
unfortunately the `WebpackBundleProject` class has no argument for setting this in the constructor